OrbitCore/FramePointerValidatorTests.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
The names of test files need to end with `Test.cpp`, not `Tests.cpp`. For example this allows clang-format to understand this is a test and reorder includes properly (in this case putting `#include "FramePointerValidator.h"` before other ones).

Reply:
Okay, I was looking at the ElfFileTests.cpp... will rename this one, too.